src/material-experimental/mdc-slider/slider.scss

```diff
@@ -6,3 +6,8 @@
 .mdc-slider {
```

Comment:
I think that we should define an `mat-mdc-slider` class and target it instead. This selector could conflict with other MDC-based sliders on the page.